portal/components/tokenSelector/tokenList.tsx

```diff
@@ -1,24 +1,27 @@
 'use client'
 
-import { useVirtualizer } from '@tanstack/react-virtual'
 import { Card } from 'components/card'
 import { Drawer } from 'components/drawer'
 import { CloseIcon } from 'components/icons/closeIcon'
+import { MagnifyingGlassIcon } from 'components/icons/magnifyingGlassIcon'
 import { SearchInput } from 'components/inputText'
 import { Modal } from 'components/modal'
 import { useDebounce } from 'hooks/useDebounce'
+import { useTopTokensToHighlight } from 'hooks/useTopTokensToHighlight'
 import { useUserTokenList } from 'hooks/useUserTokenList'
 import { useWindowSize } from 'hooks/useWindowSize'
 import partition from 'lodash/partition'
 import { useTranslations } from 'next-intl'
-import { type JSX, useRef, useState } from 'react'
+import { useState } from 'react'
 import { Token as TokenType } from 'types/token'
 import { type Chain, isAddress, isAddressEqual } from 'viem'
 
+import { List } from './list'
 import { NoTokensMatch } from './noTokensMatch'
-import { Token } from './token'
+import { TokenListSkeleton } from './tokenListSkeleton'
+import { TokenQuickSelect } from './tokenQuickSelect'
 
-const isCustomToken = (userTokenList: TokenType[], token: TokenType) =>
+export const isCustomToken = (userTokenList: TokenType[], token: TokenType) =>
```

Comment:
There's some circular reference here. It may not cause issues now, but I've seen this causing problems in some very particular scenarios, and it's tricky to track, so it's better to prevent it from happening.

This file `portal/components/tokenSelector/tokenList.tsx` imports `import { List } from './list'`. But then, that file imports `import { isCustomToken } from './tokenList'`, which is this file.